Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: help
```

Comment:
Is the Makefile only used for local rendering? Or is it also necessary for CI processes to work properly?

Currently, we don't have a Makefile in the project, so I would prefer to either migrate all commands to the Makefile or to move the docs commands to composer.json.

Reply:
It is only used for the local rendering, however the issue with composer commands is, that you need php to run composer and many local machines run php in a docker container. Then composer is trying to execute docker in docker. Therefore I would not advise to add it to the composer.json. You could, however add the command to the runTests.sh if you want. You can also omit the Makefile and ppl can execute the docker or podman command directly. However we found that a Makefile is very helpful to a large number of contributors and we use it all over the official documentation. We also advise it to all Extensions. And also there is really no harm in having a Makefile. You might want to add shortcuts for the runTests.sh in future